udffsck/log.c

```diff
@@ -0,0 +1,303 @@
+/*
+ * log.c
+ *
+ * Copyright (c) 2017    Vojtech Vladyka <vojtch.vladyka@gmail.com>
+ * All rights reserved.
```

Comment:
I'm not lawyer, but `All rights reserved` is needed? According to wikipedia it is obsolete https://en.wikipedia.org/wiki/All_rights_reserved

Reply:
It was taken from existing udftools sources, just changed name and year. 
Same is here for example: https://github.com/pali/udftools/blob/master/cdrwtool/main.c

Reply:
I know, but I rather do not change existing old files.

Reply:
Ok. Is somewhere more up-to-date header which should be used?

Reply:
You can reuse header from e.g. udflabel or udfinfo... all those are new files.

Reply:
Updated here: 0e976cc